packages/firestore/test/unit/local/schema_migration.test.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * Copyright 2018 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { expect } from 'chai';
+import { IndexedDbMutationQueue } from '../../../src/local/indexeddb_mutation_queue';
+import { IndexedDbPersistence } from '../../../src/local/indexeddb_persistence';
+import {
+  ALL_STORES,
+  createOrUpgradeDb,
+  V1_STORES
+} from '../../../src/local/indexeddb_schema';
+import { Deferred } from '../../../src/util/promise';
+
+const INDEXEDDB_TEST_DATABASE = 'schemaTest';
+
+function deleteDb() {
```

Comment:
If you add your return type, you'll likely find out you have a bug. :-)

Reply:
Done, and then replaced with SimpleDb.delete()